site/2016/11/22/doctrine-mongodb-release-1.4.0.rst

```diff
@@ -0,0 +1,89 @@
+Doctrine MongoDB 1.4.0
+======================
+
+We are happy to announce the immediate availability of Doctrine MongoDB
+Abstraction Layer `1.4.0 <https://github.com/doctrine/mongodb/releases/tag/1.4.0>`__.
+
+Passing context options to the driver
+-------------------------------------
+
+With this release it's possible to driver options to the connection class, which
+will then be passed on to the MongoDB driver. For example, to pass a stream
+context with SSL context options, you could use the following code snippet:
+
+.. code-block:: php
+
+    $context = stream_context_create([
+        'ssl' => [
+            'allow_self_signed' => false,
+        ]
+    ]);
+    $connection = new \Doctrine\MongoDB\Connection(null, [], null, null, ['context' => $context]);
+
+Passing multiple expressions to logical operators
+-------------------------------------------------
+
+The ``addAnd``, ``addNor`` and ``addOr`` methods in the query and aggregation
+builders now take multiple expression objects. Instead of having to call the
+method repeatedly, you need to only call it once now:
```

Comment:
"you may call it once with multiple arguments". I suppose the old style still works, but saying "you need to call it once now" made it sound like we're breaking BC.